cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/service/GcpResourceNameService.java

```diff
@@ -149,14 +149,17 @@ private String gcpSubnetResourceName(Object[] parts) {
         return subnetName;
     }
 
+    @SuppressWarnings("checkstyle:MagicNumber")
```

Comment:
do not ignore it with checkstylle justt use a constantt instead

Reply:
it is intended to be a number since this is a false positive. The number 3 is not a magic one and does not need an explanation. Following the logic we should provide constants in all other `checkArgs`. Inspection is very effective but stupid - it lets numbers -1,0,1,2 to pass but others trigger it. Sometimes -1,0,1,2 *are* magic numbers and sometimes number 3 or 255 are *not*. I believe this is the case then a method argument is self-explanatory and requires no constant introduction.